scripts/convex.ganache.ts

```diff
@@ -2,13 +2,14 @@ import * as assert from 'assert';
 
 const chalk: any = require('chalk');
 import * as ethers from 'ethers';
-import Ganache from 'ganache';
+import Ganache, {ProviderOptions} from 'ganache';
 
 const url = process.env.ETH_RPC_URL || 'mainnet';
-const deleteCache = process.env.CLEAR_CACHE && Number(process.env.CLEAR_CACHE) === 1;
+const deleteCache = process.env.CLEAR_CACHE && Number(process.env.CLEAR_CACHE) === 1 || false;
```

Comment:
Just sanity checking this was accidentally left in?

```suggestion
const deleteCache = process.env.CLEAR_CACHE && Number(process.env.CLEAR_CACHE) === 1;
```

Reply:
Without this, the type of `deleteCache` is `boolean | "" | undefined`, this makes it a boolean.